src/js/components/calendar.jsx

```diff
@@ -60,10 +103,17 @@ export default class Calendar extends React.Component {
     const {
       timeslots,
       initialDate,
+      onMonthChange
```

Comment:
This prop is not being used here, I only see it in `this._onMonthChange`, so it's an unused var.